src/main/groovy/de/jansauer/printcoverage/PrintCoverageTask.groovy

```diff
@@ -4,13 +4,23 @@ import org.gradle.api.DefaultTask
 import org.gradle.api.GradleException
 import org.gradle.api.provider.Property
 import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
 import org.gradle.api.tasks.TaskAction
 
 class PrintCoverageTask extends DefaultTask {
 
   @Input
+  @Optional
```

Comment:
Optional inputs with defaults handled in the task to allow custom task definitions extending this task with minimal configuration